### PR TITLE
Add overview functionality

### PR DESCRIPTION
**Pull request by ryan1604**

### CODE REVIEW

**Comment by ryan1604** [comment]
dont merge yet

**Review by NeoMinWei — DISMISSED** [review]
deeply nested code from line 37 and 49 of overview command
maybe can implement rounding where user inputs value of 3d.p.?
other than that LGTM

**Review by wwweert123 — APPROVED** [review]
LGTM, Maybe in the AddCashFlowCommand, you can create a method in UI to print the remaining budget or the exceeded budget instead of using showMessage

**Review by NeoMinWei — CHANGES_REQUESTED** [review]
Very good in abstracting out the Ui elements
However, the code in lines 37 and 49 are STILL DEEPLY NESTED

**Comment by ryan1604** [comment]
> Very good in abstracting out the Ui elements However, the code in lines 37 and 49 are STILL DEEPLY NESTED

I believe 2-3 layers of nesting is fine

**Comment by ryan1604** [comment]
> LGTM, Maybe in the AddCashFlowCommand, you can create a method in UI to print the remaining budget or the exceeded budget instead of using showMessage

Good idea! I will look into it

**Review by NeoMinWei — APPROVED** [review]

File: src/main/java/seedu/financialplanner/list/Budget.java [hunk 1]
```diff
@@ -31,6 +33,16 @@ public static double getCurrentBudget() {
         return currentBudget;
     }
 
+    public static String getCurrentBudgetString() {
+        DecimalFormat decimalFormat = new DecimalFormat("####0.00");
+        return decimalFormat.format(Cashflow.round(currentBudget, 2));
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/list/Budget.java:38]: good job on using rounding function to make the value more accurate when displayed in 2d.p.

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**